convert/convert_test.go

```diff
@@ -0,0 +1,264 @@
+package convert
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/kong/deck/file"
+	"github.com/kong/deck/utils"
+	"github.com/kong/go-kong/kong"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_validConversion(t *testing.T) {
+	type args struct {
```

Comment:
nit: `args` feels a bit superfluous to me, I see this as a pattern in many tests but I don't see it as providing value over just inlining arguments in `tests`

Not a blocker or action item, just a scathing remark that I'd whisper if we were doing code review sitting in one room :joy:

Reply:
This piece is auto-generated. 🤷🏾

Reply:
Interesting - what generator are you using?

Reply:
https://github.com/cweill/gotests via https://github.com/buoto/gotests-vim